MovieDB/MovieDB/Scenes/Dashboard/Cells/Movie Item/MovieItemTableViewCell.swift

```diff
@@ -0,0 +1,39 @@
+import UIKit
+import RxSwift
+import RxCocoa
+import Reusable
+import SDWebImage
+
+final class MovieItemTableViewCell: UITableViewCell, NibReusable {
+    @IBOutlet private weak var movieImageView: UIImageView!
+    @IBOutlet private weak var movieNameLabel: UILabel!
+    @IBOutlet private weak var movieReleaseDateLabel: UILabel!
+    @IBOutlet private weak var movieAverageRateLabel: UILabel!
+    @IBOutlet weak var bookmarkButton: UIButton!
+    @IBOutlet weak var starButton: UIButton!
```

Comment:
If you want to subscribe the action of the button, you can declare the signal similar PublishSubject on ViewModel, and onNext when user tap on button. Everyone want to know when user tap on button, they will subscribe the signal according to ViewModel.